web/interfaces.js

```diff
@@ -107,14 +107,20 @@ class IPDFLinkService {
  */
 class IPDFHistory {
   /**
-   * @param {Object} params
+   * @param {Object} options
+   * @param {*} options.fingerprint
```

Comment:
Why `*` here, since it should be a string!?

Reply:
Ah thanks, I wasn't sure about the type so I left it as `*`. Will change!